std/assembly/string.ts

```diff
@@ -578,15 +585,15 @@ import { idof } from "./builtins";
           // monkey patch
           store<u16>(codes + (j << 1), c - 26);
         } else {
-          let index = -1;
+          let index = -1 as usize;
```

Comment:
Reminds me that we once had the `"Conversion from type '{0}' to '{1}' will require an explicit cast when switching between 32/64-bit.": 201` diagnostic. Unrelated to this issue, but perhaps we should report that again?

Reply:
Will be great. More stricter type cast diagnostics - less surprises in the future)